contracts/starknet/lib/felt_to_uint256.cairo

```diff
@@ -0,0 +1,10 @@
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
```

Comment:
redundant imports lines 1 and 3

Reply:
makes me think about https://github.com/snapshot-labs/sx-core/issues/97 :)